geode-core/src/distributedTest/java/org/apache/geode/internal/cache/tier/sockets/ClientServerMiscDUnitTest.java

```diff
@@ -129,7 +130,9 @@ public void testPingWrongServer() {
     PingOp.execute(pool, new ServerLocation(NetworkUtils.getServerHostName(), PORT1), server2ID);
     // if the ping made it to server2 it will have the client's ID in its health monitor
     server2.invoke(() -> {
-      assertEquals(1, ClientHealthMonitor.getInstance().getClientHeartbeats().keySet().size());
+      await("For heartbeat to be received").timeout(Duration.ofMinutes(1))
```

Comment:
Please use the default timeout (5 minutes) set by `GeodeAwaitility` instead: `await().untilAsserted()` (without the `timeout`).

Reply:
I thought 5 minutes were too much for this case but if its ok for you, then I can change it, no problem.

Reply:
I agree in that 5 minutes might be "too much" for this test, but it's the current standard and there are only a handful of tests (8 I think) that overrides it, so I think is better to keep using the defaults for everything new.
As a side note, the poll occurs every 100 milliseconds anyway, so the condition should be met way sooner than 5 minutes.